apps-devstg/us-east-1/base-certificates/config.tf

```diff
@@ -2,19 +2,22 @@
 # AWS Provider Settings       #
 #=============================#
 provider "aws" {
-  region                  = var.region
-  profile                 = var.profile
-  shared_credentials_file = "~/.aws/${var.project}/config"
+  region  = var.region
+  profile = var.profile
+  # comment this 2 lines for aws sso enable
+  # shared_credentials_files = ["~/.aws/${var.project}/credentials"]
+  # shared_config_files      = ["~/.aws/${var.project}/config"]
```

Comment:
@rodriguez-matias Matu are this lines still present here but commented? In that case let's just remove them :) Thanks